dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -44,8 +44,8 @@ ENV NJOBS="2" \
 ENV COMPILER="4.09.0"
 
 # Common OPAM packages
-ENV BASE_OPAM="zarith.1.11 ounit2.2.2.3" \
-    CI_OPAM="ocamlgraph.1.8.8 cppo.1.6.8" \
+ENV BASE_OPAM="zarith.1.12 ounit2.2.2.6" \
```

Comment:
The `BASE_OPAM` version of required dependencies (here `zarith`) should stay in sync with the documented minimal required version, so that we ensure that we do not break compatibility accidentally. So here, I would keep `zarith.1.11`, unless there is a reason for needing 1.12, in which case, `INSTALL.md` and `coq-core.opam` should be updated.

Reply:
Indeed, not sure why I bumped this, AFAICT `zarith` 1.11 and 1.12 are identical tho.

Reply:
Fixed.